ajax/libs/arrive/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "arrive",
+  "filename": "minified/arrive.min.js",
+  "version": "2.3.1",
+  "description": "arrive.js provides events to watch for DOM elements creation and removal. It makes use of Mutation Observers internally.",
+  "repository": {
+    "type": "git",
+    "url": "git://git@github.com/uzairfarooq/arrive.git"
```

Comment:
How about `git+https://github.com/uzairfarooq/arrive.git`?

I can use `git clone git://github.com/uzairfarooq/arrive.git` and `git clone https://github.com/uzairfarooq/arrive.git` to clone the repo.

But I can't clone the repo via `git clone git://git@github.com/uzairfarooq/arrive.git`